tests/test_jx/test_deep_ops.py

```diff
@@ -820,9 +820,7 @@ def test_aggs_on_parent_and_child(self):
                     {"o": 1, "v": NULL, "s": 0},
                     {"o": 2, "v": "b", "s": 1},
                     {"o": 2, "v": "c", "s": 0},
-                    {"o": 2, "v": NULL, "s": 1},
-                    {"o": NULL, "v": "b", "s": 0},
-                    {"o": NULL, "v": "c", "s": 0}
+                    {"o": 2, "v": NULL, "s": 1}
```

Comment:
If `Null`'s are not there, may be we can change the dimensions for the cube to not include   `Null`'s. I tried this approach in [https://github.com/mozilla/jx-sqlite/pull/74](https://github.com/mozilla/jx-sqlite/pull/74). Making database to return those `Null` slices, makes many other tests to fail and that seemed wrong also. May be I am wrong. Please review #74.
Thank you